server/stream.go

```diff
@@ -1207,6 +1226,41 @@ func (mset *Stream) internalSendLoop() {
 	}
 }
 
+func (mset *Stream) internalDecLoop() {
+	mset.mu.RLock()
+	c := mset.client
+	if c == nil {
+		mset.mu.RUnlock()
+		return
+	}
+	s := c.srv
+	decq := mset.decq
+	name := mset.config.Name
+	mset.mu.RUnlock()
+
+	// Warn when internal send queue is backed up past 75%
+	warnThresh := 3 * msetSendQSize / 4
+	warnFreq := time.Second
+	last := time.Now().Add(-warnFreq)
+
+	for {
+		if len(decq) > warnThresh && time.Since(last) >= warnFreq {
+			s.Warnf("Jetstream internal decrement pending queue > 75%% for account: %q stream: %q", c.acc.Name, name)
```

Comment:
Should we capture the c.acc.Name at the top of function? Is this client an internal client that never will have c.acc changed/etc.. even during reload?

Reply:
name won't change.

Reply:
Not the name of the account, I meant will `c.acc` ever be set to nil/reset during a config reload?

Reply:
Sorry, no acc will not change either.